# The topic of LaserScan messages to subscribe to for segmentation.
scan_topic: scan

# The number of points a seed segment should contain.
# This value should be LESSER than "segment_min_points" value.
seed_segment_points: 10

# The number of points a full line segment should contain.
# This value should be GREATER than "seed_segment_points" value.
segment_min_points: 6

# The maximum distance (metre) from a seed segment point to the seed segment line.
point_to_line_threshold_m: 0.03

# The maximum distance (metre) from a seed segment point to the predicted point on the seed segment.
point_to_point_threshold_m: 0.1

# The maximum angle (rad) difference between two segments to consider them collinear.
collinear_threshold_rad: 0.1

# The frequency at which to compute the lines and publish.
update_frequency: 10.0

min_line_length_m: 0.3